### DISCUSSION TITLE
Is there a better way to organize this?

### DISCUSSION THREAD

**Discussion by j0hnm4r5** [j0hnm4r5/awesome-creative-technology#41] [Ideas]
The current method is a little unwieldy and primarily consists of a massive "Creative Technology" section and a bunch of smaller adjacent ones (like for Museums, Art Collectives, Architecture firms, etc.).

Is there a better way to organize all this? Tags?

**Comment by j0hnm4r5** [comment]
@JohnENoonan if you have any thoughts, I'd love to hear them.

**Comment by raphdag01** [comment]
Hello, why not separate medium makers (technologists) from content producers (writers, directors) ?
Then, filter by technology and application fields (museum, theatre, web, etc..) ?

**Reply by raphdag01** [reply]
BTW I would like to add my studio : we imagine and engineer, from blue sky to premiere day, live immersive experiences (Show, Theatre, Retail, Hotel, Exhibitions, Theme Parks). But I really don't get the way to suggest an update !

**Reply by j0hnm4r5** [reply]
Can you give me an example of a few places that might fall into each side of that, from the current list?

**Reply by j0hnm4r5** [reply]
@raphdag01 you can add a new submission by adding a new object in the appropriate section of `index.ts` and submitting it as a PR. Or alternatively submit an issue with all the necessary info and I will add it when I get the chance.

**Reply by raphdag01** [reply]
For example : Adirondack is a Design & Build company. They help in making the IP live. But they don't write the IP. On the other hand, Dera Lee has a strong value proposition in writing the narrative. The breakdown of the list could also be derivated from the methodolgy of design : concept, schematic, production, fabrication, commissionning, operation.

**Comment by EdwardDeaver** [comment]
Hi John, 
I'm not sure about the subcategories but I think one thing could be making the locations plain text so CMD+F/Control+F can search the page.

**Reply by j0hnm4r5** [reply]
Oh that's a great callout; I didn't realize those SVGs had broken that. I'll make that update real quick.

**Reply by EdwardDeaver** [reply]
Thank you!

**Reply by j0hnm4r5** [reply]
All done. Not as pretty, but a lot more useful.
